contracts/dtoken/tests/sim/withdraw.rs

```diff
@@ -392,3 +394,62 @@ fn scenario_withdraw_error_transfer() {
         view_balance(&controller, Supply, user.account_id(), dtoken.account_id());
     assert_eq!(user_balance, 20, "Balance should be 20");
 }
+
+#[test]
+fn scenario_view_accounts() {
+    // TODO remove in future if we will make sure it works properly
+    let (dtoken, controller, utoken, user) = supply_borrow_withdraw_fixture();
+
```

Comment:
You should also check when the user profile doesn't exists flow

Reply:
i passed the account bob.near which is not supplied or borrowed any so there no response for him